src/main/java/frc/robot/subsystems/intake/Intake.java

```diff
@@ -1,18 +1,34 @@
 package frc.robot.subsystems.intake;
 
+import com.revrobotics.CANSparkLowLevel.MotorType;
+import com.revrobotics.CANSparkMax;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.constants.RobotConstants.IntakeConstants;
 
 public class Intake extends SubsystemBase {
-  /** Creates a new ExampleSubsystem. */
-  public Intake() {}
+  private final CANSparkMax m_rollerMotor; // Intake roller motor
```

Comment:
nit: prefer 2 spaces in between code and comments on the same line, i.e.
```
private final CANSparkMax m_rollerMotor;  // Intake roller motor
```

Reply:
That is in conflict with spotless